app/assets/src/components/utils/report.js

```diff
@@ -0,0 +1,140 @@
+import ThresholdMap from "./ThresholdMap";
```

Comment:
I had several more suggestions for the code below, but you are just moving code for now, right? 
If so, you can ignore the below comments.

Reply:
Yes, just moving code. I'll take care of the camelcase.